capstone/src/main/java/com/google/sps/AssociationAnalysis.java

```diff
@@ -3,20 +3,24 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.HashMap;
+import opennlp.tools.stemmer.PorterStemmer;
 
 /** Calculates association scores from a list of sentiments per entity mention */
 public class AssociationAnalysis {
 
   HashMap<String, AssociationResult> res;
```

Comment:
You can declare these fields as private to the AssociationAnalysis class to maintain encapsulation.

Reply:
Fixed this!